### PR TITLE
all done

### PR DESCRIPTION
**Pull request by machakelvin**
This pull request addresses the bug described in https://github.com/udsm-dhis2-lab/iCareConnect/issues/228. The bug involves rectifying the error where the system attributes test results to the ordering physician rather than the laboratory technician who processes them.

**Changes**
<h1>name : Bahati Kelvin Protas</h1> 

<h3>**Regno:  2021-04-00597**</h3>

 
<h4> change made:</h4>
 i changed some lines in [ui/src/app/shared/components/patient-history/patient-history.component.html](url)

### CODE REVIEW

**Comment by machakelvin** [comment]
This pull request addresses the bug described in https://github.com/udsm-dhis2-lab/iCareConnect/issues/228. The bug involves rectifying the error where the system attributes test results to the ordering physician rather than the laboratory technician who processes them.


**changes**
name : bahati kelvin protas 
regno: 2021-04-00597
change made: i changed some lines in [ui/src/app/shared/components/patient-history/patient-history.component.html](url)

File: ui/src/app/shared/components/patient-history/patient-history.component.html [hunk 1]
```diff
@@ -27,39 +26,32 @@
...
           </span>
         </ng-template>
-        <app-patient-history-data
-          [visit]="visit"
-          [forms]="params?.customForms"
-          [FacilityDetails]="params?.facilityDetails"
-          [currentUser]="params?.currentUser"
-          [generalPrescriptionOrderType]="params?.generalPrescriptionOrderType"
-          [prescriptionArrangementFields]="
-            params?.prescriptionArrangementFields
```
Comment (DEBORAGABONE) [inline ui/src/app/shared/components/patient-history/patient-history.component.html:37]: <app-patient-history-data
  [visit]="visit"
  [forms]="params?.customForms"
  [FacilityDetails]="params?.facilityDetails"
  [currentUser]="params?.currentUser"
  [generalPrescriptionOrderType]="params?.generalPrescriptionOrderType"
  [prescriptionArrangementFields]="params?.prescriptionArrangementFields"
  [labTechnician]="params?.labTechnician">
</app-patient-history-data>

**Comment by Ramadhani202208** [comment]
There is no pull request link seen

**Comment by machakelvin** [comment]
Good afternoon

we made a commit and sent a pull request, I don't know where exactly the
problem is

**Comment by Ramadhani202208** [comment]
Where is the pull request link?

**Comment by machakelvin** [comment]
the request looks like it's closed, I can't send the request again

If possible to send screenshots of what we did I can make it.

**Comment by Ramadhani202208** [comment]
Who closed? Nothing has to do with the screen shot so far. Please contact Dr. Leonard or Mr. Vincent Minde.

**Comment by machakelvin** [comment]
Good afternoon
Here is the link of the repository used to fix bugs for both assignment 1
and 2
 https://github.com/RosemaryCharles/iCareConnect.git

**Pull request closed without merging**